contracts/common/UnstructuredStorage.sol

```diff
@@ -0,0 +1,35 @@
+/*
+ * SPDX-License-Identitifer:    MIT
+ */
+
+pragma solidity ^0.4.18;
+
+
+contract UnstructuredStorage {
+    // keccak256("uint256.initializationBlock"), used by Initializable
+    bytes32 public constant initializationBlockPosition = 0xd9120073d934f0c287a3a735b193740486dbba88578fd8f3cbb2b9a9b654e7ce;
```

Comment:
Why not move to the `Initializable` contract directly? Same comment on the naming of the key as for the other storage values in `AppStorage`.